### PR TITLE
Cleaner 4XX & 5XX error presentation to user.

### PR DESCRIPTION
**Pull request by cnrudd**

### CODE REVIEW

File: exception/Exception.js [hunk 1]
```diff
@@ -15,37 +15,32 @@ import {isString} from 'lodash';
  */
 export class Exception {
 
+    /**
+     * Create and get back a Javascript Error object
+     * @param {(Object)} cfg - properties to add to the Error object
```
Comment (amcclain) [inline exception/Exception.js]: Did we mean this to be `{(Object|string)}`?

File: exception/ExceptionHandler.js [hunk 2]
```diff
@@ -115,4 +117,10 @@ export class ExceptionHandler {
     static sessionExpired(exception) {
         return exception && exception.httpStatus === 401;
     }
+
+    static cleanStack(exception) {
+        // statuses of 0, 4XX, 5XX are server errors, so the javascript stack
+        // is irrelevant and potentially misleading
+        if (/^[045]/.test(exception.httpStatus)) delete exception.stack;
```
Comment (amcclain) [inline exception/ExceptionHandler.js]: That's great - I had been noticing this and am glad we cleaned it up here.  (Assuming that in general we weren't getting useful info here in terms of what code path triggered the call to the server.)

File: exception/Exception.js (hunk 1 above)
Comment (cnrudd) [reply exception/Exception.js]: Yes. I've corrected this in the develop branch.

File: exception/ExceptionHandler.js (hunk 2 above)
Comment (cnrudd) [reply exception/ExceptionHandler.js]: Correct.  The stack was starting with the line that threw the error as a response to the bad server response.

File: exception/Exception.js (hunk 1 above)
Comment (amcclain) [reply exception/Exception.js]: Thanks much!

**Pull request merged**